logs_ingest/mapping.py

```diff
@@ -49,7 +49,12 @@
             resource_type = resource_type_to_me_type["resourceType"].lower()
             category = resource_type_to_me_type.get("category", "").lower()
             key = ",".join(filter(None, [resource_type, category]))
-            dt_me_type_mapper.update({key: resource_type_to_me_type["meType"]})
+            me_type = resource_type_to_me_type["meType"]
```

Comment:
[optional] I think metric_type would be more readable than me_type

Reply:
me = monitored entity

Reply:
As you can see there's a problem with naming here